var/spack/repos/builtin/packages/jsonnet/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Jsonnet(MakefilePackage):
+    """A data templating language for app and tool developers based on JSON"""
+
+    homepage = "https://jsonnet.org/"
+    git = "https://github.com/google/jsonnet.git"
+
+    maintainers = ["jcpunk"]
+
+    version("develop", branch="master")
+    version("0.18.0", commit='34419d2483927ceb17cd506cad77c3c2a96e7b8c')
+    version("0.17.0", commit='ed11b012a4e1487727c595380bff8866d40f2529')
+
+    conflicts('%gcc@:5.4.99', when='@0.18.0:')
+
+    variant('python', default=False,
+            description='Provide Python bindings for jsonnet')
+    depends_on('py-setuptools', type=('build', 'run'), when='+python')
```

Comment:
Setuptools is probably only needed at build-time. Also need build deps on pip and wheel

Reply:
I think I've gotten this sorted.